_technical/dynamic-secrets.md

```diff
@@ -1,28 +1,26 @@
 ---
 title: Dynamic Secrets
-description: Description part 1
+description: Use a specific tool for managing and abstracting secret data from your applications.
 layout: pattern
 ---
 
-![Dynamic Secrets]({{ site.baseurl }}/assets/images/Generic.png)
+![Dynamic Secrets]({{ site.baseurl }}/assets/images/cs.png)
 
-Description part 2
+The team is building applications using distributed microservices architecture. The secrets required by multiple services and environments need to be managed in a manner that doesnt negatively impact agility or security.
 
-## Context
+## In This Context
 
 Building a distributed, microservices architecture. Many services require passwords, keys, tokens or other sensitive data which cannot be stored in plain text.
 
-## Problem
-
 Teams tend to end up with one of two sets of problems:
 
-### High Security - Low Agility
+##### High Security - Low Agility
 
 - All credentials are managed (manually) be a security team. Nobody usually knows where this team actually sits, there only interface is an obscure ticket filing system.
```

Comment:
- All credentials are managed (manually) be a security team. Nobody usually knows where this team actually sits, the only interface is an obscure ticket filing system.